include/sys/uzfs_zvol.h

```diff
@@ -53,9 +53,7 @@ typedef struct metaobj_blk_offset {
  */
 typedef struct zvol_rebuild_data {
 	/* mutex to synchronize io tree operation */
-	kmutex_t io_tree_mtx;
 	uint64_t rebuild_bytes;
-	avl_tree_t *incoming_io_tree;   /* incoming io tree */
 } zvol_rebuild_data_t;
```

Comment:
should this structure be removed? (there is just one number inside and stale comment)

Reply:
Thanks for pointing out this.
```zvol_rebuild_data_t``` was meant to hold all rebuild related variables. So better to keep as it is.

As of now, there are two variables we are using for rebuild purpose ```rebuild_bytes``` and ```zv_rebuild_status```. I will move ```zv_rebuild_status``` to ```zvol_rebuild_data_t``` structure.

I will also remove stale comment.